src/net/posix-stack.cc

```diff
@@ -427,8 +427,17 @@ posix_server_socket_impl::accept() {
     return _lfd.accept().then([this] (std::tuple<pollable_fd, socket_address> fd_sa) {
         auto& fd = std::get<0>(fd_sa);
         auto& sa = std::get<1>(fd_sa);
-        auto cth = _lba == server_socket::load_balancing_algorithm::connection_distribution ?
-                _conntrack.get_handle() : _conntrack.get_handle(ntoh(sa.as_posix_sockaddr_in().sin_port) % smp::count);
+        auto cth = [this, &sa] {
+            switch(_lba) {
```

Comment:
:+1: for improving the code readability.

Reply:
i'd suggest add a space after `switch`.